docs/connecting/java.md

```diff
@@ -67,8 +67,7 @@ To get started with JanusGraph in Java:
 4.  Create a `GraphTraversalSource` which is the basis for all Gremlin traversals:
 
     ```java
-    Graph graph = EmptyGraph.instance();
-    GraphTraversalSource g = graph.traversal().withRemote("conf/remote-graph.properties");
+    GraphTraversalSource g = AnonymousTraversalSource.traversal().withRemote("conf/remote-graph.properties");
```

Comment:
Please don't use the `AnonymousTraversalSource` directly. The idea was to create it with just `traversal()` which requires to statically import `AnonymousTraversalSource`. See [this section in the TinkerPop reference docs](http://tinkerpop.apache.org/docs/current/reference/#connecting-gremlin-server) where it's shown for the different languages or [in the upgrade docs](http://tinkerpop.apache.org/docs/current/upgrade/#_anonymoustraversalsource) where it was explained when it was introduced in TinkerPop.

Reply:
I basically update all use cases again for this one. Do you like to check it again?